python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -143,16 +143,26 @@ class PsfexPsfDeterminerTask(measAlg.BasePsfDeterminerTask):
     ConfigClass = PsfexPsfDeterminerConfig
 
     def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
-        """Determine a PSFEX PSF model for an exposure given a list of PSF candidates
-
-        @param[in] exposure: exposure containing the psf candidates (lsst.afw.image.Exposure)
-        @param[in] psfCandidateList: a sequence of PSF candidates (each an lsst.meas.algorithms.PsfCandidate);
-            typically obtained by detecting sources and then running them through a star selector
-        @param[in,out] metadata  a home for interesting tidbits of information
-        @param[in] flagKey: schema key used to mark sources actually used in PSF determination
-
-        @return psf: a meas.extensions.psfex.PsfexPsf
+        """Determine a PSFEX PSF model for an exposure given a list of PSF candidates.
+
+        Parameters
+        ----------
+        exposure: `lsst.afw.image.Exposure`
+            Exposure containing the PSF candidates
+        psfCandidateList: iterable of `lsst.meas.algorithms.PsfCandidate`
+            Sequence of PSF candidates typically obtained by detecting sources and then running them through a 
+            star selector
+        metadata: metadata, optional
```

Comment:
Maybe type is `dict` here?

Reply:
I don't think it can be a dict since later on it calls metadata.set('key', value) if it's not None, and I haven't found any code using this task at all (let alone with metadata).

Reply:
`lsst.daf.base.PropertySet`?